### DISCUSSION TITLE
Support for deleting old from Collection from Trakt when sync new collection from Plex to Trakt

### DISCUSSION THREAD

**Discussion by axez85** [Taxel/PlexTraktSync#549] [Ideas]
I have updated PlexTraktSync to version 0.15.15 and now seen a potential for this.
Because of that i want to change my question.

Are there any plans to provide the option to remove old collections from Trakt when sync with Plex. For example inside the config file by writing a line to delete movies and series that do not exist in Plex when sync takes place.
(to keep from new collection from Plex and remove movies that is not in collection in Plex from Trakt)

Keep upp the good work!

**Comment by pterisaur** [comment]
Agreed this is needed! For most things, the "append" sync method works great — I don't want to remove ratings or status for items I have in Trakt but no longer have in Plex. However, the collection is an exception; if I remove an item from Plex, I want it to be removed from my Trakt collection. Is it possible to have different sync methods for different aspects of the sync?

**Reply by simonc56** [reply]
You can do as explained in https://github.com/Taxel/PlexTraktSync/issues/374 workaround
